CHANGELOG.md

```diff
@@ -1,5 +1,35 @@
 # idb-pconnector change log
 
+# 1.1.0
+
+- feat: Add support for enableNumericTypeConversion on DBPool functions ([#97](https://github.com/IBM/nodejs-idb-pconnector/pull/97))
+
+- feat: Add support for bindParameters ([#92](https://github.com/IBM/nodejs-idb-pconnector/pull/92))
+   - bindParameters() deprecates bindParam() and bind()
+
+- refactor: Statement tests ([#78](https://github.com/IBM/nodejs-idb-pconnector/pull/78))
+  - No longer use deprecated Statement with implicitly connection
+  - No longer use QIWS.QCUSTCDT from insert tests
+  - Add after/afterEach hooks to delete data inserted by test cases
+  - Close out open statement and connection handles in each test
+
+- docs: Add select bindParam example ([#101](https://github.com/IBM/nodejs-idb-pconnector/pull/101))
+  Signed-off-by: Jhonny <jhonnattanriverarivera@gmail.com>
+
+-  fix: Add package-lock.json ([#86](https://github.com/IBM/nodejs-idb-pconnector/pull/86))
+
+- refactor: Deprecate implicitly creating a connection
+
+- docs: for enableNumericTypeConversion() [#54](https://github.com/IBM/nodejs-idb-pconnector/issues/54)
+
+  Co-Authored-By: Kevin Adler <kadler@us.ibm.com>
```

Comment:
Probably don't need the Signed-off-by since that's more of a copyright thing for us to accept it. Probably don't need the Co-Authored-By either since there's a link to the PR if people care (and we don't list the main author either).

Reply:
OK I will remove that co-authored lines